package.json

```diff
@@ -14,7 +14,8 @@
     "dev": "webpack-dev-server --mode development --host=0.0.0.0 --disable-host-check",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "wba": "npm run build:prod && npx webpack-bundle-analyzer dist/stats.json"
+    "wba": "npm run build:prod && npx webpack-bundle-analyzer dist/stats.json",
+    "cypress:run": "cypress run"
```

Comment:
remember to include running eslint in cypress folder, you can either modify

`"test:fix-lint": "eslint test cypress --fix"`

or use a new script

`"cypress:test:fix-int": "eslint cypress --fix"`

Reply:
I get this error running eslint

`
/Users/josepita/code/wikipedia-preview/cypress/plugins/index.js
  19:1  error  'module' is not defined  no-undef`

but I was checking the file from the kaios repo and it looks exactly the same...

Reply:
I can suggest you do one of the below:

updating it to es6

```diff
- module.exports = ( on, config ) => {
+ export default ( on, config ) => {
```

or adding `module` in as the global var

```js
globals: {
    ...
    module: true
},
```

Reply:
the first option is not recognised by Cypress, fails the run.

the second one, where should I add it?

Reply:
`.eslintrc.js`